mpas_analysis/shared/timekeeping/Date.py

```diff
@@ -117,7 +118,7 @@ def to_timedelta(self):
                              "instead of to_timedelta")
 
         days = 365*self.years + self._monthsToDays(self.months) + self.days
```

Comment:
Assumes that there are no leap years, which is not general and probably needs fixed in #82 for the case where leap years may be allowed for MPAS.